pkg/tfgen/installation_docs.go

```diff
@@ -106,24 +110,50 @@ func writeFrontMatter(providerDisplayName string) string {
 // .NET: Pulumi.foo
 // Java: com.pulumi/foo
 // ****
-func writeInstallationInstructions(goImportBasePath, displayName, pkgName string) string {
+//
+// A dynamically bridged provider receives the following instead:
+// ****
+// ## Generate Provider
+//
+// The Foo provider must be installed as a Local Package by following the instructions for Any Terraform Provider: (link)
+// ```bash
+// pulumi package add terraform-provider org/foo
+// ```
+// ****
+func writeInstallationInstructions(goImportBasePath, displayName, pkgName, ghOrg, sourceRepo string) string {
 	// Capitalize the package name for C#
 	capitalize := cases.Title(language.English)
 	cSharpName := capitalize.String(pkgName)
 
-	return fmt.Sprintf(
-		"## Installation\n\n"+
-			"The %[1]s provider is available as a package in all Pulumi languages:\n\n"+
-			"* JavaScript/TypeScript: [`@pulumi/%[2]s`](https://www.npmjs.com/package/@pulumi/%[2]s)\n"+
-			"* Python: [`pulumi-%[2]s`](https://pypi.org/project/pulumi-%[2]s/)\n"+
-			"* Go: [`%[4]s`](https://github.com/pulumi/pulumi-%[2]s)\n"+
-			"* .NET: [`Pulumi.%[3]s`](https://www.nuget.org/packages/Pulumi.%[3]s)\n"+
-			"* Java: [`com.pulumi/%[2]s`](https://central.sonatype.com/artifact/com.pulumi/%[2]s)\n\n",
-		displayName,
-		pkgName,
-		cSharpName,
-		goImportBasePath,
-	)
+	var instructions string
+	if strings.Contains(sourceRepo, "pulumi") {
+		instructions = fmt.Sprintf(
```

Comment:
You can't return the `if` expression (since it isn't an expression), but we should return in it.
```suggestion
	if strings.Contains(sourceRepo, "pulumi") {
		return fmt.Sprintf(
```